### PR TITLE
Added Disclaimer

### PR DESCRIPTION
**Pull request by ccsouthard**
Added simple Disclaimer section to sub-footer

### CODE REVIEW

**Comment by ccsouthard** [comment]
The conflict could be spaces vs. tabs?

File: view/layout.marko [hunk 1]
```diff
@@ -42,6 +42,8 @@ html
       div.footer-column
         h3 -- About Us
         p -- Bella Surfaces, Inc. has over 20 years of combined experience in installing custom surfaces in Kitchens, Bathrooms, Playrooms and Commercial facilities throughout the New England area. As seen in "This Old House" Newton, MA Project December 2017. Bella Surfaces, Inc. specializes in countertops, bathroom vanities, bars, fireplaces, reception areas, commercial surfaces, and many, many more. We strive to offer the best customer service in combination with our courteous professional installation teams.
+      `h4 -- Disclaimer
```
Comment (notslang) [inline view/layout.marko]: there's not supposed to be a backtick before the h4

**Comment by notslang** [comment]
This is a weird disclaimer to add. I don't think I've ever seen a company website with one like this.

**Review by ccsouthard — COMMENTED** [review]
fixed typo

File: view/layout.marko (hunk 1 above)
Comment (ccsouthard) [reply view/layout.marko]: Fixed

**Pull request closed without merging**